test/fixtures/full/base/main.tf

```diff
@@ -42,10 +42,9 @@ resource "google_project" "test" {
   count = local.n
```

Comment:
I hit a race condition where `google_storage_bucket.test` failed to apply due to this project not being ready. I think we should attach all resources to the project created in `test/setup` and only use this for testing project IAM bindings. Both `google_project_service` resources should also be removed as they are already enabled on the base project.

Reply:
This should resolve the error in CI.

Reply:
Moved SA creation to the "preparation" step, now CI tests runs well